penguin/cli/__init__.py

```
@@ -47,35 +47,7 @@ def get_cli_app():
-except ImportError:
-    # CLI dependencies not available (minimal install)
-    __all__ = ["get_cli_app"]
-
-
-
-# Import journal commands to register them
-try:
-    from .journal_commands import journal_command
-except ImportError:
-    pass  # journal commands might not be available
+__all__ = ["cli_app", "get_cli_app", "journal_command", "PenguinCLI"]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)